american-journal-of-archaeology.csl

```diff
@@ -300,7 +317,7 @@
       <if variable="title" match="none"/>
       <else-if type="thesis"/>
       <else>
-        <text variable="genre" text-case="capitalize-first" prefix=". "/>
```

Comment:
If not from here, where are you getting the period from?

Reply:
I believe it's at the end of the title, which precedes the event, which is itself part of the issue.